### PR TITLE
[Improvement] Optimize server startup log

### PR DESCRIPTION
**Pull request by ruanwenjun**
## Purpose of the pull request

- Optimize the server startup log, e.g `config format`, `Rpc server`, `log patten of class`.
- Remove unused code.

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterConfig.java [hunk 1]
```diff
@@ -163,23 +163,26 @@ public void validate(Object target, Errors errors) {
...
-        log.info("Master config: stateWheelInterval -> {} ", stateWheelInterval);
-        log.info("Master config: maxCpuLoadAvg -> {} ", maxCpuLoadAvg);
-        log.info("Master config: reservedMemory -> {} ", reservedMemory);
-        log.info("Master config: failoverInterval -> {} ", failoverInterval);
-        log.info("Master config: killApplicationWhenTaskFailover -> {} ", killApplicationWhenTaskFailover);
-        log.info("Master config: registryDisconnectStrategy -> {} ", registryDisconnectStrategy);
-        log.info("Master config: masterAddress -> {} ", masterAddress);
-        log.info("Master config: masterRegistryPath -> {} ", masterRegistryPath);
-        log.info("Master config: workerGroupRefreshInterval -> {} ", workerGroupRefreshInterval);
+        String config = "\n****************************Master Configuration**************************************";
```
Comment (caishunfeng) [inline dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterConfig.java]: Is it better to use `StringBuilder`?

File: dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/TaskConstants.java [hunk 2]
```diff
@@ -327,45 +303,13 @@ private TaskConstants() {
...
-     */
-    public static final String ORG_POSTGRESQL_DRIVER = "org.postgresql.Driver";
-    public static final String COM_MYSQL_CJ_JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
-    public static final String ORG_APACHE_HIVE_JDBC_HIVE_DRIVER = "org.apache.hive.jdbc.HiveDriver";
-    public static final String COM_CLICKHOUSE_JDBC_DRIVER = "com.clickhouse.jdbc.ClickHouseDriver";
-    public static final String COM_DATABEND_JDBC_DRIVER = "com.databend.jdbc.DatabendDriver";
-    public static final String COM_ORACLE_JDBC_DRIVER = "oracle.jdbc.driver.OracleDriver";
-    public static final String COM_SQLSERVER_JDBC_DRIVER = "com.microsoft.sqlserver.jdbc.SQLServerDriver";
-    public static final String COM_DB2_JDBC_DRIVER = "com.ibm.db2.jcc.DB2Driver";
-    public static final String COM_PRESTO_JDBC_DRIVER = "com.facebook.presto.jdbc.PrestoDriver";
```
Comment (caishunfeng) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/TaskConstants.java:355]: Don't need these constant anymore?

**Comment by caishunfeng** [comment]
Please add some pr description.

File: dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/TaskConstants.java (hunk 2 above)
Comment (ruanwenjun) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/TaskConstants.java:355]: Yes, this already removed into `DataSource` plugin module.

File: dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterConfig.java (hunk 1 above)
Comment (ruanwenjun) [reply dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterConfig.java]: Done

**Comment by ruanwenjun** [comment]
> Please add some pr description.

Done

**Review by caishunfeng — APPROVED** [review]
LGTM

**Pull request merged**